Tools/Schema.NET.Updater/Program.cs

```diff
@@ -22,9 +24,23 @@ private static async Task Main()
 
         Console.WriteLine(Resources.SavingTo, SchemaJsonDestinationFilePath);
 
-        using var fileStream = File.Open(SchemaJsonDestinationFilePath, FileMode.Create);
+        var fullSavePath = Path.Combine(GetProjectDirectory() ?? string.Empty, SchemaJsonDestinationFilePath);
+        using var fileStream = File.Open(fullSavePath, FileMode.Create);
         await stream.CopyToAsync(fileStream).ConfigureAwait(true);
 
         Console.WriteLine(Resources.UpdateComplete);
     }
+
+    private static string? GetProjectDirectory()
```

Comment:
Tried with Linq, don't think its much more readable.

```c#
private static string? GetProjectDirectory() =>
    Attribute
        .GetCustomAttributes(typeof(Program).Assembly)
        .OfType<AssemblyMetadataAttribute>()
        .Where(x => x.Key == ProjectDirectoryAttributeKey)
        .Select(x => x.Value)
        .FirstOrDefault();
```

Reply:
Yeah, was going to do similar but I think the plain foreach comes out slightly ahead in readability.